src/containers/item-details/item-details.tsx

```diff
@@ -25,17 +25,16 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
         <div className="row g-5">
           <div
             className="col-lg-7 col-md-12 col-sm-12"
-            style={{ height: '100vh', position: 'sticky' }}
+            style={{ height: '100vh', position: 'relative' }}
           >
-            <Sticky>
-              <img
-                src={`${
-                  item.image.url
-                }?img-width=${740}&img-height=${560}&img-fit=${'crop'}&img-quality=${85}`}
-                alt="NFT_portfolio"
-                style={{ borderRadius: '20px' }}
-              />
-            </Sticky>
+            <Image
+              src={`${
+                item.image.url
+              }?img-width=${740}&img-height=${560}&img-fit=${'crop'}&img-quality=${85}`}
```

Comment:
Are you calling calling the image using next instead of calling it directly from Pinata?

Reply:
Hey guys I just wanted fix the issue, I didn't see more :(

Reply:
Don't worry! Tomorrow we can check it out together

Reply:
I didn't changes in the way to bring images